### PR TITLE
Update example code.

### PR DESCRIPTION
**Pull request by thelonelyvulpes**

### CODE REVIEW

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs [hunk 1]
```diff
@@ -21,1127 +21,1077 @@
...
+            using var session = Driver.Session();
+            var persons = session.ExecuteRead(tx => tx.Run("MATCH (a:Person) RETURN a.name AS name").ToList());
+            return persons.Sum(person => session.ExecuteWrite(tx =>
+            {
+                var result = tx.Run("MATCH (emp:Person {name: $person_name}) " +
+                                    "MERGE (com:Company {name: $company_name}) " +
+                                    "MERGE (emp)-[:WORKS_FOR]->(com)",
+                    new {person_name = person["name"].As<string>(), company_name = companyName});
+                result.Consume();
+                return 1;
```
Comment (RichardIrons-neo4j) [inline Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:620]: probably just being dumb but why does this just return a hardcoded 1?

**Review by RichardIrons-neo4j — COMMENTED** [review]
All looks fine to me but somebody else should probably check too. I did put one note but it was just a question and doesn't need a change.

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs (hunk 1 above)
Comment (thelonelyvulpes) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:620]: I don't really know, it was in the example before.

Comment (AndyHeap-NeoTech) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:620]: Looks like it has always been that way. I assume the use of the method doesn't make use of the return value?

Comment (thelonelyvulpes) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:620]: it's used on line 632 to validate this was called twice.

Comment (RichardIrons-neo4j) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:620]: Haha. Thanks for the answer!

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs [hunk 2]
```diff
@@ -21,1127 +21,1077 @@
...
-                // Given
-                using (var driver = CreateDriverWithCustomizedSecurityStrategy(Uri, User, Password))
                 using (var session = driver.Session())
                 {
-                    // When & Then
-                    session.Run("RETURN 1").Single()[0].As<int>().Should().Be(1);
+                    session.Run("CREATE (a:Person {name: $name})", new {name});
                 }
             }
         }
```
Comment (robsdedude) [inline Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:261]: This code is not being exercised, is it?

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs [hunk 3]
```diff
@@ -21,1127 +21,1077 @@
...
+            ~DatabaseSelectionExample() => Dispose(false);
+
+            public DatabaseSelectionExample(string uri, string user, string password)
+            {
+                _driver = GraphDatabase.Driver(uri, AuthTokens.Basic(user, password));
             }
 
-            [RequireEnterpriseEdition("5.0.0", VersionComparison.GreaterThanOrEqualTo)]
-            public async void TestUseAnotherDatabaseExampleAsync()
+            public void UseAnotherDatabaseExample()
```
Comment (robsdedude) [inline Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:869]: Is this example being tested somewhere?

**Review by robsdedude — APPROVED** [review]
:sweat:

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs (hunk 2 above)
Comment (thelonelyvulpes) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:261]: Testkit runs them as part of the integration test suite.

File: Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs (hunk 3 above)
Comment (thelonelyvulpes) [reply Neo4j.Driver/Neo4j.Driver.Tests.Integration/Examples.cs:869]: yup(ln:821, 845)

**Pull request merged**